components/Common/Preview/index.tsx

```diff
@@ -37,7 +37,7 @@ const Preview: FC<PreviewProps> = ({
           src={`${basePath}/static/images/logos/js-white.svg`}
           alt="Node.js"
         />
-        <h1>{title}</h1>
+        <h2>{title}</h2>
```

Comment:
When I examined the designs, I saw that this component(`Preview`) was used in more than one place. Marking this element directly as `h1`, `h2` or `h3` may give us a disadvantage in terms of SEO. Instead, I recommend using it directly. We can pass the heading elements from above

```suggestion
        {title}
```

Reply:
Thank you! Makes total sense. It's true that adding this change will give the responsibility of passing the tag to the "outside world" of the component, which is something I tend to avoid, as someone could pass a title wrapped in a `p` or an `h1` without considering the implications, but the possible accessibility issue makes for a good reason.